docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -1604,6 +1604,67 @@ public class OidcStartup {
 For more complex setup involving multiple tenants please see the xref:security-openid-connect-multitenancy.adoc#programmatic-startup[Programmatic OIDC start-up for multitenant application]
 section of the OpenID Connect Multi-Tenancy guide.
 
+== Step Up Authentication
+
+The `io.quarkus.oidc.AuthenticationContext` annotation can be used to enforce required authentication level for the Jakarta REST resource classes and methods.
+The https://datatracker.ietf.org/doc/rfc9470/[OAuth 2.0 Step Up Authentication Challenge Protocol] introduces a mechanism for resource servers to request stronger token authentication methods.
+Consider the following example:
+
+[source,java]
+----
+package io.quarkus.it.oidc;
+
+import io.quarkus.oidc.AuthenticationContext;
+import io.quarkus.oidc.BearerTokenAuthentication;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+@BearerTokenAuthentication
+@Path("/")
+public class GreetingsResource {
+
+    @Path("hello")
+    @AuthenticationContext("myACR") <1>
+    @GET
+    public String hello() {
+        return "hello";
+    }
+
+    @Path("hi")
+    @AuthenticationContext(maxAge = 120) <2>
```

Comment:
If we have `myACR` requirement at the class level but only `maxAge` at the method level, should we inherit.

How about us making the `value` (acr) property required, and if users want, they can add it time constrained ? That would avoid some possibly tricky situation where a class level acr is set and some sub-resource method has only `maxAge` ?

Reply:
Done.